pages/settings/Settings.qml

```diff
@@ -101,6 +101,14 @@ ColumnLayout {
                     previousView.onPageClosed();
                 }
             }
+            if ((previousView == settingsLayoutView && (currentView == settingsWalletView))  ||
+                (previousView == settingsNodeView   && (currentView == settingsLayoutView || currentView == settingsWalletView)) ||
+                (previousView == settingsLogView    && (currentView != settingsInfoView)) ||
+                (previousView == settingsInfoView)) {
```

Comment:
I've been trying to find a better solution for this, ideally with a position variable, but didn't find a nice solution yet.

Reply:
Using previousIndex and currentIndex now